scripts/format.sh

```diff
@@ -30,9 +30,9 @@ find src valhalla test -type f -name '*.h' -o -name '*.cc' \
 py=$(setup_python)
 if [[ $(python3 -m pip list | grep -c "black\|flake8") -ne 2 ]]; then
   if [[ $(python3 -c 'import sys; print(int(sys.base_prefix != sys.prefix or hasattr(sys, "real_prefix")))') -eq 1 ]]; then
-    ${py} -m pip install black==22.10.0 flake8==5.0.4
+    ${py} -m pip install black==24.10.0 flake8==7.1.1
```

Comment:
there was a bug in flake8==5.0.4 that caused false positives of E231 (`missing whitespace after :`)